libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -4,12 +4,12 @@
       <img class="reading-list-item--cover" [src]="b.coverUrl" />
     </div>
     <div class="reading-list-item--details">
-      <strong class="reading-list-item--details--title">{{ b.title }}</strong>
+      <strong data-testing= "list-item-title" class="reading-list-item--details--title">{{ b.title }}</strong>
```

Comment:
There is no consistency with the spaces around the `=` sign. I would remove them.

Reply:
@mandarini: Review addressed, please take a look.